### PR TITLE
Add all assets required for Gaming Platform.

### PR DESCRIPTION
**Pull request by yoginth**
Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of watchers:
   - Count of stars:
   - Count of forks:
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@yoginth Thank you for your contribution.
I refer to https://github.com/cdnjs/cdnjs/pull/11174 and know that you what CDNJS host some files let people access your website more quickly.  
As the title `Added all assets required for Gaming Platform` I think you want add all the file into CDNJS.
But I think there are some file CDNJS already host as the followings, which under [`vendor`](https://github.com/swapifier/assets/tree/master/vendor), [`vendor/backpack`](https://github.com/swapifier/assets/tree/master/vendor/backpack), [adminlte](https://github.com/swapifier/assets/tree/master/vendor/adminlte), maybe you could use these links at first and no need to add this files again:
- **vendor**
dropify: https://cdnjs.com/libraries/Dropify
notie: https://cdnjs.com/libraries/notie
owlcarousel: https://cdnjs.com/libraries/owl-carousel
summernote: https://cdnjs.com/libraries/summernote
- **vendor/backpack**
tinymce: https://cdnjs.com/libraries/tinymce
summernote: https://cdnjs.com/libraries/summernote
select2: https://cdnjs.com/libraries/select2
pnotify: https://cdnjs.com/libraries/pnotify
nestedSortable: https://cdnjs.com/libraries/nestedSortable
ladda: https://cdnjs.com/libraries/Ladda
elfinder: https://cdnjs.com/libraries/elfinder
cropper: https://cdnjs.com/libraries/cropper
colorbox: https://cdnjs.com/libraries/jquery.colorbox
ckeditor: https://cdnjs.com/libraries/ckeditor
- **vendor/adminlte**
bootstrap: http://getbootstrap.com/getting-started/#download-cdn (I think you can use bootstrap's cdn)
- **vendor/adminlte/plugins**
bootstrap-slider: https://cdnjs.com/libraries/bootstrap-slider
chartjs: https://cdnjs.com/libraries/Chart.js
ckeditor: https://cdnjs.com/libraries/ckeditor
colorpicker: https://cdnjs.com/libraries/bootstrap-colorpicker
datatables: https://cdnjs.com/libraries/datatables
datepicker: https://cdnjs.com/libraries/bootstrap-datepicker
daterangepicker: https://cdnjs.com/libraries/bootstrap-daterangepicker
fastclick: https://cdnjs.com/libraries/fastclick
flot: https://cdnjs.com/libraries/flot
fullcalendar: https://cdnjs.com/libraries/fullcalendar
iCheck: https://cdnjs.com/libraries/iCheck
input-mask: https://cdnjs.com/libraries/jquery.inputmask
ionslider: https://cdnjs.com/libraries/ion-rangeslider
jQuery: https://cdnjs.com/libraries/jquery
jQueryUI: https://cdnjs.com/libraries/jqueryui
jvectormap: https://cdnjs.com/libraries/jvectormap
knob: https://cdnjs.com/libraries/jQuery-Knob
morris: https://cdnjs.com/libraries/morris.js
pace: https://cdnjs.com/libraries/pace
select2: https://cdnjs.com/libraries/select2
slimScroll: https://cdnjs.com/libraries/jQuery-slimScroll
sparkline: https://cdnjs.com/libraries/jquery-sparklines
timepicker: https://cdnjs.com/libraries/bootstrap-timepicker

Could you help confirm are these libraries that you need? 
If yes, I think you could use this library link first, and these files which under `vendor` that CDNJS no need to host again from this PR.
If some libraries is not you want to add from these links, maybe you could serch from CDNJS's website(https://cdnjs.com)

And there are some folders need to confirm.
- [bootstrap-iconpicker](https://github.com/victor-valencia/bootstrap-iconpicker): someone open an issue #8052, I'll help to add this.
- crud: I think this created by you, is it?
- adminlte/build: Is it created by yourself? or you add it from another library?
- adminlte/dist: Is it created by yourself?
- [bootstrap-wysihtml5](https://github.com/jhollingworth/bootstrap-wysihtml5): need to send another PR to add this library.

May I ask you what file which you created by your team or yourself that CDNJS should host? 

Or you just want CDNJS host `swapifier-site.min.css` and the file under `css`, `js` ?
Thank you.

**Comment by yoginth** [comment]
Please host only swapifier-site.min.css and some files i will auto-update latter, can i auto-update?

File: ajax/libs/swapifier/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "adminlte"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/swapifier/assets.git"
+  },
+  "npmName": "swapifier",
+  "npmFileMap": [
+    {
+      "basePath": ".",
```
Comment (sufuf3) [inline ajax/libs/swapifier/package.json]: let's use `"basePath": "",` to get the basePath  of file

File: ajax/libs/swapifier/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      "files": [
+        "**/*"
```
Comment (sufuf3) [inline ajax/libs/swapifier/package.json]: let's use `"swapifier-site*"` to get the swapifier-site.min.css

**Comment by yoginth** [comment]
however we have modified assets so please host them all!

**Comment by sufuf3** [comment]
@yoginth which assets that you have modified? All of them?

**Comment by sufuf3** [comment]
@PeterDaveHello 
I found the website `swapifier` use some of the libs from CDNJS. 
`view-source:https://friendstub.com/`

And the repo https://github.com/swapifier/assets has not enough popularity.
And there is no response since 19 Jun

What can we do for this PR?
Still waiting for the contributor?
Or close this PR?

Thank you.

**Pull request closed without merging**